web/src/components/shared/Item/item.css

```diff
@@ -0,0 +1,73 @@
+.block{
```

Comment:
please put some spaces...
more over check: https://www.nafrontendzie.pl/metodyki-css-2-bem
and change classes